src/components/activity-introduction/activity-page-links.tsx

```diff
@@ -14,18 +15,20 @@ export class ActivityPageLinks extends React.PureComponent <IProps> {
       <div className="activity-page-links" data-cy="activity-page-links">
         <div className="pages">Pages in this Activity</div>
         { this.props.activityPages.filter((page) => !page.is_hidden).map((page, index: number) => (
-          <div className="page-item" key={`index ${index}`} onClick={this.handleButtonClick(index + 1)}>
+          <div className="page-item" key={`index ${index}`} onClick={this.handleButtonClick(index + 1)} onKeyDown={this.handleButtonClick(index + 1)}>
             <span>{`${index + 1}. `}</span>
-            <span className="page-link" onClick={this.handleButtonClick(index + 1)}>{`Page ${index + 1}`}</span>
+            <span className="page-link" onClick={this.handleButtonClick(index + 1)} onKeyDown={this.handleButtonClick(index + 1)} tabIndex={0}>{`Page ${index + 1}`}</span>
           </div>
           ))
         }
-        <button className="button begin" onClick={this.handleButtonClick(1)}>Begin Activity</button>
+        <button className="button begin" onClick={this.handleButtonClick(1)} onKeyDown={this.handleButtonClick(1)}>Begin Activity</button>
       </div>
     );
   }
 
   private handleButtonClick = (page: number) => () => {
```

Comment:
Consider renaming since this function is used by the button and key events.  Maybe `handlePageChange`